CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 Mapbox welcomes participation and contributions from everyone.
 # main
+* Remove module setup driven by the mapbox-base modularization using annotations. The SDK depends directly on the MapTelemetryImpl versus the MapTelemetry interface module. ([#1693](https://github.com/mapbox/mapbox-maps-android/pull/1693))
```

Comment:
if we know it should be cherry-picked to v10.9 - perhaps create the header here already and place this in correct section? btw - is this bug fix or the feature? 🤔